### PR TITLE
Align with colab

### PR DESCRIPTION
**Pull request by Rachidrh**
- Use Percentrank transformer from LIR
- Update LIR requirements

### CODE REVIEW

**Comment by annajeannette** [comment]
- Bugfix: data werd eerst maar 1x geladen en gepreprocessed, wat ervoor zorgde dat de resultaten niet klopten. 
- ranktransformer: uit lr-benchmark gehaald, en vervangen door PercentileRankTransformer
- bugfix: Seed:None moest Seed: int = None zijn.

File: lrbenchmark/dataset.py [hunk 1]
```diff
@@ -106,10 +107,13 @@ def load(self) -> XYType:
     def get_x_y(self) -> XYType:
         if self._data is None:
             X, y = self.load()
-            if self.preprocessor:
-                X = self.preprocessor.fit_transform(X)
             self._data = (X, y)
-        return self._data
+        else:
+            (X, y) = self._data
```
Comment (rolfypma-nfi) [inline lrbenchmark/dataset.py:112]: ik zou verwachten dat als er geen preprocessor is, deze X y teruggegeven worden? Of moet er een zijn? Dan verwacht ik een error als er geladen wordt zonder. nu krijg je 'silently' een none?

Comment (annajeannette) [reply lrbenchmark/dataset.py:112]: Er moet inderdaad een preprocessor zijn, als er geen is gebruiken we 'dummy'. Als je geen waardes hebt voor preprocessor zijn er ook geen resultaten.

Comment (annajeannette) [reply lrbenchmark/dataset.py:112]: Alsnog aangepast. Ook wordt er nu een error gegeven als er voor een parameter (bvb preprocessor) geen waardes zijn ingevuld.
